caps/capabilities.go

```diff
@@ -51,9 +51,14 @@ type Repository struct {
 	caps map[string]*Capability
 }
 
+// NotFound means that a capability was not found
+type NotFound struct {
```

Comment:
Corrected but please see if you like the implementation  78589a0b550101669199deb6b78fd058595b72fd